lib/matplotlib/tests/test_backend_bases.py

```diff
@@ -325,9 +327,7 @@ def test_toolbar_home_restores_autoscale():
 def test_draw(backend):
     from matplotlib.figure import Figure
     from matplotlib.backends.backend_agg import FigureCanvas
-    test_backend = pytest.importorskip(
-        f'matplotlib.backends.backend_{backend}'
-    )
+    test_backend = importlib.import_module(f'matplotlib.backends.backend_{backend}')
```

Comment:
`backend` here is svg, ps, pdf, or pgf, which are not expected to fail (maybe they used to when Pillow was optional?)

Reply:
maybe crossed streams with the mark on the pgf parameter?